medusa/tv/series.py

```diff
@@ -1048,8 +1048,8 @@ def make_ep_from_file(self, filepath):
                     if not cur_ep:
                         raise EpisodeNotFoundException
                 except EpisodeNotFoundException:
-                    logger.error(u'{indexerid}: Unable to figure out what this file is, skipping {filepath}',
-                                 indexerid=self.indexerid, filepath=filepath)
+                    logger.warning(u'{indexerid}: Episode not found for this show. Please full update show. Skipping {filepath}',
```

Comment:
`Unable to determine episode for this file.  Please run a full update on the series.`